perl/bin/cgpVaf.pl

```diff
@@ -218,54 +211,35 @@ sub option_builder {
         $options{'tmp'}=$options{'o'}.'/tmpvaf_'.$tn_name;
     }
     if(defined $options{'a'} and ( (lc($options{'a'}) eq 'indel') || (lc($options{'a'}) eq 'snp') ) ) {
-        warn "Analysing:".$options{'a'}."\n";
+        $log->info(sprintf 'Analysing: %s', $options{'a'});
     }
     else{
-        $log->logcroak("Not a valid variant type [should be either [snp or indel]");
+        $log->logcroak('Not a valid variant type [should be either [snp or indel]');
         exit(0);
     }
-     # use annotation tags to output in tsv
-    if(!defined $options{'t'}) {
-        $options{'t'}="VD,VW,VT,VC";
-    }
-    # input alignment file extension
-    if(!defined $options{'be'}) {
-        $options{'be'}=".bam";
-    }
+
     # use PASS flag if bed only is set
     if($options{'bo'}==1) {
-        $options{'r'}= 0;
+        $options{'r'} = 0;
     }
     if(!defined $options{'r'}) {
-        $options{'r'}= 1;
+        $options{'r'} = 1;
     }
-        
+
     if($options{'a'} eq 'indel' && !defined $options{'dp'}) {
         $options{'dp'}='NR,PR';
     }
 
     if($options{'ct'}) {
-        $log->debug("Concatenation option selected, chromosome option will be set to all chromosomes");
+        $log->debug('Concatenation option selected, chromosome option will be set to all chromosomes');
         $options{'chr'}=[];
     }
```

Comment:
see comment for line 63 and 64

Reply:
Documentation states that `-ct` and `-chr` are for separate executions.

Reply:
yes, this is true